api/client.yaml

```diff
@@ -1437,6 +1523,10 @@ components:
           type: string
           example: f6eddffd
           description: accountID of funding/source account for this Organization
+        logoFile:
```

Comment:
Sorry I didn't catch this before, but the UI expects a full URI it can access. I think `logoFile` is confusing here.